cdap-ui/app/cdap/components/DatasetDetailedView/Tabs/AuditTab.js

```diff
@@ -32,12 +32,13 @@ export default function AuditTab({entity}) {
   url += `&sourceUrl=${encodedSource}`;
 
   return (
-    <div className="iframe-container">
+    <div className="embed-responsive embed-responsive-16by9">
```

Comment:
I have just added 16:9. 1:1 and 4:3 just caused too much scrolling on a regular 13" screen for an iframe. Any suggestions?